src/estimators/any_sketch.py

```diff
@@ -272,6 +306,8 @@ def __init__(self, config, random_seed=None, hash_function=HashFunction):
     self.sketch = np.zeros(
         tuple(len(i.distribution) for i in config.index_specs),
         dtype=np.int32)
+    if config.value_functions[0] == UniqueKeyFunction():
```

Comment:
This has the potential to cause problems. If we can make the FLAG_EMPTY_REGISTER == 0, then we can delete this.

Reply:
Yes, See comment above.